editor/src/clj/editor/app_view.clj

```diff
@@ -1382,7 +1403,10 @@ If you do not specifically require different script states, consider changing th
                                                  (not-empty
                                                    (g/with-auto-evaluation-context evaluation-context
                                                      (updated-build-resources evaluation-context project old-etags etags "/game.project")))]
-                                       (engine/reload-build-resources! target updated-build-resources))
+                                       (if (targets/all-launched-targets? target)
+                                         (doseq [launched-target (targets/get-all-launched-targets)]
+                                           (engine/reload-build-resources! launched-target updated-build-resources))
```

Comment:
magic of hot-reloading for all the instances! 

https://github.com/user-attachments/assets/fc9024d0-7879-48fd-ac2c-fc6e67cd3b41